cockroachcloud/cloud-api.md

```diff
@@ -390,6 +390,46 @@ Where:
 
 If the `DELETE` request was successful the client will not receive a response payload.
 
+<a id="cloud-audit-logs"></a>
+
+## Export {{ site.data.products.db }} audit logs
+
+{% include feature-phases/preview-opt-in.md %}
+
+To export audit logs for activities and events related to your {{ site.data.products.db }} organization, send a `GET` request to the `/v1/auditlogevents` endpoint. The service account associated with the secret key must have `ADMIN` [permission](console-access-management.html#service-accounts).
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request GET \
+  --url 'https://cockroachlabs.cloud/api/v1/auditlogevents?StartingFrom={timestamp},SortOrder={sort_order},limit={limit}' \
+  --header 'Authorization: Bearer {secret_key}'
+~~~
+
+Where:
+
+  - `{timestamp}` indicates the first log entry to fetch. If unspecified, `{timestamp}` is set to the beginning of time if `{sort_order}` is `ascending`, or the current time if `{sort_order}` is `descending`.
+  - `{sort_order}` is either `ascending` (the default) or `descending`.
+  - `{limit}` indicates roughly how many entries to return. It is expected that slightly more results will be returned, because if any entries exist for a timestamp, all entries for that timestamp are always fetched. It defaults to `200`.
+
+A request that includes no parameters exports roughly 200 entries are returned in ascending order, starting from the first entry for your {{ site.data.products.db }} organization.
+
+If the request was successful, the client will receive a JSON array consisting of a list of log `entries` and a `next_starting_from` field.
```

Comment:
@rgcase Would we return "next_starting_from" even when there no more results to return?

Reply:
If the results are returned in descending order (latest to earliest) then we won't return a `next_starting_from` if there are no more results, because it's impossible to ever have more results show up in the past. If we're paginating in the other direction we will return a `next_starting_from` because more events may come in the future.